xml/ha_storage_protection.xml

```diff
@@ -348,6 +343,16 @@
        </para>
       </listitem>
      </varlistentry>
+     <varlistentry>
+      <term><literal>stonith-watchdog-timeout</literal> in the CIB</term>
+      <listitem>
+       <para>
+        This timeout is set in the CIB as global cluster property. If not set
+        explicitly, it defaults to <literal>0</literal>, which is appropriate for
+        using SBD with 1-3 devices. For use of SBD in diskless mode, see FIXME
+        for more details.</para>
+      </listitem>
+     </varlistentry>
     </variablelist>
   <para>
    If you change the watchdog timeout, you need to adjust the other two timeouts
```

Comment:
Should we mention somewhere the relationship between positive timeouts and negative timeouts?

Reply:
This is mentioned in the callouts in pro.ha.storage.protect.confdiskless.